src/common/options.cc

```diff
@@ -8340,8 +8340,13 @@ std::vector<Option> get_mds_client_options() {
     .set_flag(Option::FLAG_STARTUP),
 
     Option("fuse_big_writes", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+#ifdef LIBFUSE_VERSION3
     .set_default(false)
     .set_description("big_writes is deprecated in libfuse 3.0.0"),
+#else
+    .set_default(true)
+    .set_description("enable big_writes in libfuse < 3.0.0"),
+#endif
```

Comment:
how about set this option to true by default, make ceph-fuse ignore this option if linked to libfuse3

Reply:
Yeah, sounds better.

Reply:
@ukernel Fixed it. Thanks.